.editorconfig

```diff
@@ -8,6 +8,10 @@ trim_trailing_whitespace = true
 indent_style = tab
 indent_size = 4
 
+[{*.js}]
+indent_style = tab
+indent_size = 2
```

Comment:
Should we remove `indent_size` for JS and PHP? Or at least make them consistent.

Gutenberg and core don't specify it, except for YAML files which have specific requirements.

Reply:
I think including it is worthwhile.

Anecdotally I recall a lot of the other Gutenberg core team members use 2-space tabs (a lot of code is formatted under that assumption). It makes the code visually similar to other JS... and leaving JS at 8-width tabs will make for pretty tough-to-read JSX code.

I guess if you don't include it people can set it in their editors, but that's more work for every user 🤷‍♂

Reply:
Shouldn't the `indent_size` for JS match the PHP one (4)?

Btw @tofumatt for next time, please open an issue for such work so we can prioritize and track it on the project board.

Reply:
The majority of JS projects I've worked on, especially in the React ecosystem, have been 2 space indentation (with spaces). Using a 2-character indentation width at least keeps us visually consistent with the rest of the JS ecosystem, and given the nesting required in a lot of JS/JSX, it makes the code a lot more readable.

Reply:
Also: sounds good on creating an issue; this was just something I did locally already and had sat around so I submitted, but will do for next time 😄

Reply:
Oh no, I actually checked on this and apparently [my attempt to add this to WordPress](https://github.com/WordPress/gutenberg/pull/7754) was rejected back in the day. So I suppose this isn't a thing in the WordPress JS ecosystem 😢